package.json

```diff
@@ -36,14 +39,15 @@
     "qejs": "^3.0.5",
     "ractive": "^0.6.1",
     "react": "^0.14.6",
+    "react-dom": "^0.14.7",
     "react-tools": "^0.13.0",
```

Comment:
No, it isn't. An about the failing tests, the problem is that react doesn't support anything below Node 4. I notice it after sending the pull request, so maybe this is a big drawback in order to accepting it.